danielMunteanu/src/main/java/firstpart/testclas.java

```diff
@@ -0,0 +1,24 @@
+package firstpart;
+
+
+class testclas {
+    public static <T, D> void printArray(T[] arr, D[] b) {
+        //do your printing stuff here; something like this will work
```

Comment:
what a nice comment, could you please remove it?

Reply:
removed